kubernate/resource/documents/Readme.md

```diff
@@ -0,0 +1,85 @@
+ # <img src="../images/mongodb.png" width="" height="30"> Mongodb on <img src="../images/kubernetes.png" width="" height="27"> Kubernetes
+
+## Create cluster by KinD
+***[KinD](https://kubernetes.io/blog/2020/05/21/wsl-docker-kubernetes-on-the-windows-desktop/): Kubernetes made easy in a container***. 
+<br>
+<br>
+Right now, we have Docker that is installed, configured and the last test worked fine.
+
+This is normal as we didn't enable the Docker Kubernetes cluster. So let's install KinD and create our first cluster.
```

Comment:
Can I use Minikube instead the Kind?

Reply:
I don't know. I haven't studied it much.